account_multicurrency_revaluation/wizard/print_currency_unrealized_report.py

```diff
@@ -21,12 +21,13 @@ class UnrealizedCurrencyReportPrinter(models.TransientModel):
     start_date = fields.Date(
         string="Start Date",
         help="The report will print from this Date, all the revaluated entries"
-        "created from this date. The default value will be the first day of the month.",
+        "created from this date. The default value will be the first day of the month."
+        " ",
         default=lambda self: self._default_start_date(),
     )
     end_date = fields.Date(
         string="End Date",
-        help="The report will print till this Date. The default value will be today.",
+        help="The report will print till this Date. The default value will be today. ",
```

Comment:
This one only has one line so no need space at end.
```suggestion
        help="The report will print till this Date. The default value will be today.",
```